encoder_preprocess.py

```diff
@@ -37,15 +36,27 @@ class MyFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescriptio
     parser.add_argument("-s", "--skip_existing", action="store_true", help=\
         "Whether to skip existing output files with the same name. Useful if this script was "
         "interrupted.")
+    parser.add_argument("--disable_webrtcvad_check", action="store_true", help=\
```

Comment:
too long name and too technical
use "--no_trim" instead

Reply:
Implemented. At this time, specifying --no_trim will not have an effect if webrtcvad is present. A potential point of confusion but anyone who gets that far is capable of dealing with it themselves.